packages/cms-lib/lib/logs.js

```diff
@@ -34,7 +35,7 @@ const formatStackTrace = log => {
 };
 
 const formatTimestamp = log => {
-  return `[${moment(log.createdAt).format('MM/DD/YYYY hh:mm:ss a')}]`;
+  return `[${moment(log.createdAt).format(TIMESTAMP_FORMAT)}]`;
```

Comment:
Since we don't know if users may be remotely reading/writing these files, shouldn't we be using  
`moment().toISOString()`
for portability and ISO8601?

Reply:
This is a good point. I will update both the logs and watch task to log datetime as ISOString.